### DISCUSSION TITLE
Windows

### DISCUSSION THREAD

**Discussion by k1e2n3t4** [markqvist/Reticulum#581] [Sideband]
pip fails to load sideband on windows. says no matching distribution found for kivy-deps.sdl2_dev~=0.7.0

**Comment by k1e2n3t4** [comment]
same issue on second windows machine.

**Comment by faragher** [comment]
Strange. It works for me on Windows 10. 

Is pip failing to install, or does Sideband fail to start? You can try installing the dependency directly using `pip install kivy-deps.sdl2-dev` if only for troubleshooting. PyPi lists 0.7.0 as available, even if it's targeted directly. Are you running on the Windows command line, in Cygwin, or another method?
